Jenkinsfile

```diff
@@ -382,7 +382,6 @@ pipeline{
 	agent none;
 	options {
   	checkoutToSubdirectory 'data-hub'
-  	skipStagesAfterUnstable()
```

Comment:
If we remove this. PR's will be merged even if tests fail.

Reply:
if any of the tests fail then stage('tests') will be unstable. Then post section in lines 549-554 will abort entire build with failure.

Reply:
got it. Thanks for confirming.

Reply:
> If we remove this. PR's will be merged even if tests fail.
if any of the tests fail then stage('tests') will be unstable. Then post section in lines 549-554 will abort entire build with failure.